src/js/Mixins/Keyboard.js

```diff
@@ -1,9 +1,16 @@
-const KeyboardMixins = {
+// use function to create a new mixin object for keeping isolation
+// to make it work for multiple map instances
+const createKeyboardMixins = () => ({
   _lastEvents: { keydown: undefined, keyup: undefined, current: undefined },
   _initKeyListener(map) {
     this.map = map;
     L.DomEvent.on(document, 'keydown keyup', this._onKeyListener, this);
     L.DomEvent.on(window, 'blur', this._onBlur, this);
+    // clean up global listeners when current map instance is destroyed
+    map.on('unload', () => {
+      L.DomEvent.off(document, 'keydown keyup', this._onKeyListener, this);
+      L.DomEvent.off(window, 'blur', this._onBlur, this);
+    });
   },
```

Comment:
I think a named function would be better and cleaner. Also `once` should be enough
```suggestion
    map.once('unload', this_unbindKeyListenerEvents, this);
  },
  _unbindKeyListenerEvents() {
      L.DomEvent.off(document, 'keydown keyup', this._onKeyListener, this);
      L.DomEvent.off(window, 'blur', this._onBlur, this);
  },
```